packages/databyss-ui/components/PDFDropZone/PDFDropZoneManager.js

```diff
@@ -35,9 +36,71 @@ const viewStyles = () => ({
 const StyledView = styled(View, viewStyles)
 
 // methods
+const buildSource = data => {
```

Comment:
Arg `data` and return value should have the shape defined by the `Source` interface: https://github.com/databyss-org/databyss/blob/master/packages/databyss-services/interfaces/Block.ts#L36